test/tools/ossfuzz/protoToYul.cpp

```diff
@@ -593,6 +593,12 @@ void ProtoConverter::visit(UnaryOp const& _x)
 		return;
 	}
 
+	if (op == UnaryOp::BLOBHASH && !m_evmVersion.hasBlobHash())
+	{
+		m_output << dictionaryToken();
```

Comment:
Doesn't this mean that you should add `blobhash` to `test/tools/ossfuzz/config/solidity.dict` and `test/tools/ossfuzz/config/strict_assembly.dict`, since it is also available in Yul? Or these dictionaries are not used and instead you generate them using the proto files `yulProto` and `solProto`? If so, shouldn't we update the readme to mention that? Or maybe I'm missing something.

Reply:
Actually, adding the blobhash keyword to the dictionary may make a program invalid if EVMVersion < cancun. But in principle you are right.

Reply:
I'm approving then. But please rebase ;)

Reply:
Thank you, rebased.